src/SFA.DAS.AANHub.Domain/Common/Constants.cs

```diff
@@ -1,25 +1,29 @@
 ﻿using System.Diagnostics.CodeAnalysis;
 
-namespace SFA.DAS.AANHub.Domain.Common
+namespace SFA.DAS.AANHub.Domain.Common;
+
+[ExcludeFromCodeCoverage]
+public static class Constants
 {
-    [ExcludeFromCodeCoverage]
-    public static class Constants
+    public static class MembershipStatus
+    {
+        public const string Pending = nameof(Pending);
+        public const string Cancelled = nameof(Cancelled);
+        public const string Live = nameof(Live);
+        public const string Withdrawn = nameof(Withdrawn);
+        public const string Deleted = nameof(Deleted);
+    }
+
+    public static class MembershipUserType
     {
-        public static class MembershipStatus
-        {
-            public const string Pending = nameof(Pending);
-            public const string Cancelled = nameof(Cancelled);
-            public const string Live = nameof(Live);
-            public const string Withdrawn = nameof(Withdrawn);
-            public const string Deleted = nameof(Deleted);
-        }
+        public static readonly string Apprentice = UserType.Apprentice.ToString();
+        public static readonly string Employer = UserType.Employer.ToString();
+        public static readonly string Partner = UserType.Partner.ToString();
+        public static readonly string Admin = UserType.Admin.ToString();
+    }
 
-        public static class MembershipUserType
-        {
-            public static readonly string Apprentice = UserType.Apprentice.ToString();
-            public static readonly string Employer = UserType.Employer.ToString();
-            public static readonly string Partner = UserType.Partner.ToString();
-            public static readonly string Admin = UserType.Admin.ToString();
-        }
+    public static class Pagination
```

Comment:
Rename the class to CalendarEvents as paging if required else where may not have same constant value

Reply:
ok